pyhelper_utils/exceptions.py

```diff
@@ -1,7 +1,10 @@
+from typing import Optional
+
+
 class CommandExecFailed(Exception):
-    def __init__(self, name, err=None):
+    def __init__(self, name: str, err: Optional[str] = None) -> None:
```

Comment:
please use 

`from __future__ import annotations`  >> must be import at first line.
`str | None = None`

Reply:
Ok, I see what you have done in a previous PR. Will do